CMakeLists.txt

```diff
@@ -6,11 +6,13 @@ option(BUILD_apps "Build application programs" ON)
 option(BUILD_test "Build test programs" OFF)
 option(BUILD_PYTHON_BINDINGS "Build python bindings" OFF)
 
-if(${CMAKE_SYSTEM_PROCESSOR} STREQUAL "aarch64")
+if(${CMAKE_SYSTEM_PROCESSOR} STREQUAL "aarch64" OR ${CMAKE_SYSTEM_PROCESSOR} STREQUAL "arm64")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mtune=native")
+  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -mtune=native")
 else()
   add_definitions(-msse -msse2 -msse3 -msse4 -msse4.1 -msse4.2)
-  set(CMAKE_C_FLAGS "-msse -msse2 -msse3 -msse4 -msse4.1 -msse4.2")
-  set(CMAKE_CXX_FLAGS "-msse -msse2 -msse3 -msse4 -msse4.1 -msse4.2")
+  set(CMAKE_C_FLAGS "-msse -msse2 -msse3 -msse4 -msse4.1 -msse4.2 -march=native")
```

Comment:
Because PCL installed via apt-get causes segfaults with ```march=native``` (Possibly, the CI failure is caused by this flag), I would like to make this optional like:
https://github.com/koide3/ndt_omp/blob/0852c95360d1b0d29745e7eae7a57c0950de695b/CMakeLists.txt#L7

Reply:
Actually i think we should use `PCL_DEFINITIONS` here instead of manually specifying any flags, as any PCL client project has to be compiled with the same flags to prevent such crashes.

Reply:
Added that in 672e71f36336ec15031b10285ec8160cba6293ca